csharp/Oracle/Oracle.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo.SmartContract.Framework" Version="3.0.0-preview5" />
+    <PackageReference Include="Neo.SmartContract.Framework" Version="3.0.0-rc1" />
```

Comment:
I prefer to use the last one

Reply:
`3.0.0-rc1` is the last one, right?

Reply:
`3.0.0-CI00251` ?

Reply:
I think the examples should be more stable. It is difficult for normal users to use the packages on myget.